index.html

```diff
@@ -1186,6 +1186,29 @@ <h4 class="text-xl font-serif">Contact</i>
     <div id='map' class="map hidden lg:block">
     </div>
 
+    <div id="filters-disclaimer"
+      class="absolute w-[488px] top-9 right-[76px] flex h-16 pl-4 pr-3 py-3 bg-yellow-500 rounded-lg shadow items-center gap-3 hidden text-gray-700 text-xs leading-tight">
+      <div class="w-[424px]">
+        <span
+          class="leading-tight">Please
+          note that all filters apply to the results in the list, while
+        </span>
+        <span
+          class="font-semibold">only
+          land use type and intervention filters</span><span>
+          apply to the map.</span>
+      </div>
```

Comment:
I think the sentence would be better like this:

```suggestion
        <span
          class="leading-tight">Please
          note that all filters apply to the results in the list, while
        </span>
        <span
          class="font-semibold">only
          the land use type and intervention filters</span><span>
          apply to the map.</span>
      </div>
```